.github/workflows/release.yml

```diff
@@ -49,10 +52,13 @@ jobs:
     - name: Import cached image
       if: ${{ steps.cache.outputs.cache-hit }}
       run: |
-        if -f [[ "$CORE_CACHEFILE" ]]
+        OUTFILE="$BUILD_CACHE/$CORE_CACHEFILE"
+        if [[ -f "$OUTFILE" ]]
```

Comment:
I am pedantic, but generally prefer -e (exists) or even better -r (readeable) as a check instead of making the assumption that if it is a file I can read it (permissions, symlinks, etc)

Reply:
I need it to be a file, not some other readable entity like a directory or device file.  So `-f` is the minimal correct check. Because this file is created completely within a closed ecosystem (created by actions, saved in the cache, and restored by actions) all running as the same account with admin privileges, all filesystem entities are implicitly readable if they exist. I chose to forgo doing the second `-r` check because it should never fail, and because that would require implementing additional error reporting to indicate that the file exists but is not readable.  That extra reporting already exists in the `docker import` command.  The is-a-file check also should exist in `docker import`, but I wanted to handle the situation where the cache is not already populated - which also should never happen, but I'm apparently half-pedantic. 🤣